### PR TITLE
fix broken unit test

### PR DESCRIPTION
**Pull request by kyrregjerstad**
Fixes the unit test that broke in: https://github.com/NoroffFEU/agency.noroff.dev/pull/878

### CODE REVIEW

File: src/js/api/posts/create.test.js [hunk 1]
```diff
@@ -1,14 +1,20 @@
 import { create } from './create.js';
-import { apiBaseFetch } from '../apiBaseFetch.js';
 
-jest.mock('../apiBaseFetch.js');
 
 describe('create', () => {
+  
+  global.fetch = jest.fn(() => {});
```
Comment (Olekrr) [inline src/js/api/posts/create.test.js:6]: Global fetch is being called inside the describe block, I think this leads to timing issues.
It should be moved to the top level to ensure its set up before the "create" function is imported and used.

**Review by Olekrr — CHANGES_REQUESTED** [review]
Just gonna do a review since I've had a major hand in this.

File: src/js/api/posts/create.test.js (hunk 1 above)
Comment (kyrregjerstad) [reply src/js/api/posts/create.test.js:6]: Good point, i've moved the mock outside of the describe block!

**Review by Olekrr — APPROVED** [review]

**Comment by Olekrr** [comment]
There's currently a new rule preventing PR's from being merged if they dont pass all checks.
Because of this PR's are piling up and no one can merge anything atm.

As such I have added the linting fix to this PR as well so we can merge it asap.
Sorry to leave your QA out of this, but we have to get things going again.

**Comment by kyrregjerstad** [comment]
no probs, just get the ball rolling again!

**Pull request merged**